clang/test/Lexer/minimize_source_to_dependency_directives_invalid_import.m

```diff
@@ -0,0 +1,6 @@
+// RUN: %clang_cc1 -verify -print-dependency-directives-minimized-source %s 2>&1
+
+import <invalid.h>
+#import "invalid.h" // expected-error {{unexpected extra tokens inside declaration}}
```

Comment:
This is valid for MSVC for COM related stuff and is also dependency relevant there. I think it would be best to just forward this to the frontend to deal with.

Reply:
In what language mode? Ultimately, the minimized output does generate an assertion failure or a crash (depending on how the compiler was built) because the minimizer is attempting to reason about invalid code.
It's not very clear to me how to avoid this scenario in a way that appeases all expected use cases.